### ISSUE TITLE
/*! autoprefixer: on */ does not seem to work

### ISSUE THREAD

**Issue by dariusrosendahl** [postcss/autoprefixer#920]
I recently created https://github.com/postcss/autoprefixer/issues/919 and it seems I was looking in the wrong direction:

One of our SCSS files uses the ```/*! autoprefixer: off */``` and ```/*! autoprefixer: on */``` comments.

```/*! autoprefixer: off */``` works fine! Turning it on again is another matter:

This does not work:
```
/*! autoprefixer: off */
::-webkit-input-placeholder {
	opacity: 0;
}

:-moz-placeholder,
::-moz-placeholder {
	opacity: 0;
}

:-ms-input-placeholder {
	color: transparent;
}
/*! autoprefixer: on */
```
It breaks prefixing for the code below it.

This works:

```
::-webkit-input-placeholder {
	/*! autoprefixer: off */
	opacity: 0;
}

:-moz-placeholder,
::-moz-placeholder {
	/*! autoprefixer: off */
	opacity: 0;
}

:-ms-input-placeholder {
	/*! autoprefixer: off */
	color: transparent;
}
```

Whatever we do, we can't turn on autoprefixer again when using it around a codeblock.

**Comment by ai** [comment]
Off/on applies doesn't work in stream. It applies to whole block. In this case you switched off and switched on.

I will add warning for this cases since this mistake become popular.

**Comment by ai** [comment]
Warning was added c754a9c

**Comment by ai** [comment]
Released in 7.1.6.
